pom.xml

```
@@ -14,11 +14,11 @@
     </parent>
     <groupId>io.jenkins.plugins</groupId>
     <artifactId>opentelemetry</artifactId>
-    <version>${revision}${changelist}</version>
+    <version>${revision}.${changelist}</version>
     <packaging>hpi</packaging>
     <properties>
-        <revision>2.19.1</revision>
-        <changelist>-SNAPSHOT</changelist>
+        <revision>2</revision>
```

Given 2.19.1 is greater than 2, this must be at least 3 to prevent RC collision.
```suggestion
        <revision>3</revision>
```

---

yep, good catch